### ISSUE TITLE
Support older versions of ss without "--no-header"

### ISSUE THREAD

**Issue by rcj** [magit/ssh-agency#17]
Some distributions ship older versions of ss that don't have the "--no-header" flag and hence ssh-agency will fail.

Are there any objections against omitting the flag and simply discard the first line? That way it should work irregardless of the version of ss.

```modified   ssh-agency-20180508.26/ssh-agency.el
@@ -203,9 +203,9 @@ ssh-agency always finds the agent without consulting this file."
 (cl-defun ssh-agency-find-socket-from-ss (&key glob regexp)
   "Use `ss' to find an ssh-agent socket matching GLOB and/or REGEXP."
   (catch 'socket
-    (dolist (sock-line (apply #'process-lines
-                              "ss" "--no-header" "--listening" "--family=unix"
-                              (if glob (list "src" glob))))
+    (dolist (sock-line (cdr (apply #'process-lines
+                                   "ss" "--listening" "--family=unix"
+                                   (if glob (list "src" glob)))))
       (let* ((socket (nth 4 (split-string sock-line)))
              (status (and (or (null regexp) (string-match-p regexp socket))
                           (ssh-agency-socket-status socket))))```

**Comment by npostavs** [comment]
No objections, I just didn't realize that there were such versions.

**Comment by npostavs** [comment]
Now fixed by #18.
